stdlib/public/core/Bool.swift

```diff
@@ -109,11 +109,20 @@ public struct Bool: Sendable {
   ///   the new random value.
   /// - Returns: Either `true` or `false`, randomly chosen with equal
   ///   probability.
-  @inlinable
-  public static func random<T: RandomNumberGenerator>(
+  @_alwaysEmitIntoClient
+  public static func random<T: RandomNumberGenerator & ~Copyable>(
```

Comment:
Can't we use `@_preInverseGenerics` here? This will strip any `~Copyable` from the mangled name to get us the previous mangled name.

Reply:
Similarly for almost everything in this PR.

Reply:
Yeah, that's probably right in this case.